app/api/custom/events.py

```diff
@@ -1,24 +1,30 @@
 from flask import Blueprint, jsonify
-from sqlalchemy import or_
-from app.api.helpers.permissions import jwt_required
-from app.models.event import Event
-from app.models.session import Session
-from sqlalchemy import asc, func
+from sqlalchemy import asc, func, or_
+
 from app.models import db
+from app.models.session import Session
 
 events_routes = Blueprint('events_routes', __name__, url_prefix='/v1/events')
 
-@events_routes.route('/<string:event_identifier>/sessions/dates')
-@jwt_required
-def get_dates(event_identifier):
-    event=Event.query.filter_by(identifier=event_identifier).first()
-    dates = (
-            list(map(str, list(zip(
-                *db.session.query(func.date(Session.starts_at))
-                .distinct()
-                .filter(Session.event_id==event.id, Session.starts_at != None, or_(Session.state == 'accepted', Session.state == 'confirmed'))
-                .order_by(asc(func.date(Session.starts_at)))
-                .all()
-            ))[0]))
+
+@events_routes.route('/<int:event_id>/sessions/dates')
+def get_dates(event_id):
+    dates = list(
+        map(
+            str,
+            list(
+                zip(
+                    *db.session.query(func.date(Session.starts_at))
+                    .distinct()
+                    .filter(
+                        Session.event_id == event_id,
+                        Session.starts_at != None,
+                        or_(Session.state == 'accepted', Session.state == 'confirmed'),
+                    )
+                    .order_by(asc(func.date(Session.starts_at)))
+                    .all()
+                )
+            )[0],
+        )
```

Comment:
Too much nesting, it was better before

```python
event=Event.query.filter_by(identifier=event_identifier).first()
dates = (
        list(map(str, list(zip(
            *db.session.query(func.date(Session.starts_at))
            .distinct()
            .filter(Session.event_id==event.id, Session.starts_at != None, or_(Session.state == 'accepted', Session.state == 'confirmed'))
            .order_by(asc(func.date(Session.starts_at)))
            .all()
        ))[0]))
)
```

Reply:
Pre commit changed this. whenever i try to commit pre-commit changes my code to this.

Reply:
But you were able to commit this before. 

If you format your code like this and break any line longer than 120 chars, it will not change it

Reply:
> But you were able to commit this before. 
> 
> If you format your code like this and break any line longer than 120 chars, it will not change it



Previously i commited on github directly. I will again do that.